### ISSUE TITLE
[Question] is it possible to filter certain words during search?

### ISSUE THREAD

**Issue by barretgao** [twintproject/twint#789]
Just have a general question: is it possible to filter certain words when you use the search method? for example, I want to not get covid-19 related tweets that contain words like "democracy" "violence" etc.Thanks

**Comment by BonfaceKilz** [comment]
Yes you can. You'd use twitter's advanced search and your query would look something like: `twint -s "-democracy -violence"`. To construct a query, you could use: https://twitter.com/search-advanced

**Comment by BonfaceKilz** [comment]
@barretgao I think you can go ahead and close this. Wdyt?
